modules/exploits/multi/http/playsms_uploadcsv_exec.rb

```diff
@@ -0,0 +1,202 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+    def initialize(info = {})
+        super(update_info(info,
+        'Name' => 'PlaySMS 1.4 - import.php, Authenticated  CSV file upload Remote Code Execution',
```

Comment:
Generally module names do not contain the software version.

Suggest:
* `PlaySMS import.php Authenticated CSV File Upload Remote Code Execution`
* `PlaySMS import.php Authenticated CSV File Upload Code Execution`
* `PlaySMS import.php Authenticated CSV File Upload`

It's an exploit module, so code execution is implied.